Update on template

kinsDeveloper opened the pull request:
Added files, made changes to templates and regions, included header
information for stickiness

Commit:
Update on template

Added files, made changes to templates and regions, included header
information for stickiness

Toddses commented on src/scss/main.scss:
```diff
@@ -24,12 +24,21 @@
+// typography - SPECIFIC TO IES
+@import url('https://fonts.googleapis.com/css?family=Lato');
+@import url('https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300');
+
```
Font imports into the `_typography` partial plz.

Toddses commented on src/scss/_global.scss:
```diff
@@ -9,3 +12,17 @@
+  border: 2px outset color("red");
+  border-radius: 15px;
+  font-size: 1.5em;
+}
```
We can discuss, but in general I've found it's best to have a `_buttons` partial and tuck away button styling into that. Sometimes the buttons styling can get more complex, and also makes it easier to find if it's in it's own file.

Be sure to add the additional import for the buttons partial to main.scss.

Toddses commented on README.md:
```diff
@@ -106,6 +108,10 @@ For icons, also use SVGs, for many of the same reasons. Something like Font Awes
+## Template Files
+
+Any Drupal 7 Theme requries at minimum of two template files, but due to the significant differences between the site and the front page, a separate template file is created to change the location of the headers and remove the hero image usage only on the front page. It allows greater flexibility in customizing that section as well while negating the need to use the configuration site information ...
+
```
Title this section `Templates` and list out the provided templates and their purpose.

Toddses commented on src/js/sticky-header.js:
```diff
@@ -0,0 +1,17 @@
+            $("#header").removeClass("sticky");
+        }
+    });
+})(jQuery);
```
I'm going to be a little nitpicky here, but for JS use single quotes unless doubles are specifically required. Also provide a variable here for the scrollTop switch value, so it's slightly easier to find and change.

Also the spacing is weird. Should be 2 spaces for indentation..

Commit:
Change request reply

Made the asked for changes, added buttons component scss, updated global
and typography (specifically to include the button font which I forgot
to add last time)

Commit:
change response take 2

Made changes and updates

Toddses commented on src/js/sticky-header.js:
```diff
@@ -0,0 +1,18 @@
+
+  // Adding background color to header for scrolling
+  $(window).on('scroll', function(){
+    var topHeight = 100; 
```
this needs to be outside of the scroll event handler. otherwise it gets set on every iteration and that's not really what we're after.

Commit:
rearrange

It's rearranged :)

Commit:
take 4

cache
